Dockerfile

```diff
@@ -1,6 +1,6 @@
 # specify the dependency versions (can be overriden with --build_arg)
-ARG quilc_version=1.16.0
-ARG quicklisp_version=2019-11-30
+ARG quilc_version=1.17.0
+ARG quicklisp_version=2020-02-18
```

Comment:
does this quicklisp have the new magicl, or is that not required?

Reply:
It doesn't have it, but there's no getting around that. I just updated it for the sake of updating it.

I'm going to set up a rigetti quicklisp distribution that has all the latest quilc/qvm/rpcq/magicl/etc. so that we don't have this issue in future.